subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/InstrumentationTestsTask.kt

```diff
@@ -121,6 +121,13 @@ abstract class InstrumentationTestsTask @Inject constructor(
     @OutputDirectory
     val output: DirectoryProperty = objects.directoryProperty()
 
+    @Internal
+    private val verdictFile = objects.fileProperty().convention(output.file("verdict.json"))
+
+    @get:Internal
```

Comment:
Can it be an `@Output`? Will it give something to wire data?

Reply:
I don't know. Verdict content has already been put to the `@OutpupDirectory`